vtrace/platforms/win32.py

```diff
@@ -1135,7 +1139,7 @@ def getDebugPrivileges():
     dbgluid = LUID()
     token = HANDLE(0)
 
-    if not advapi32.LookupPrivilegeValueA(0, "seDebugPrivilege", addressof(dbgluid)):
+    if not advapi32.LookupPrivilegeValueA(0, "seDebugPrivilege".encode('utf-8'), addressof(dbgluid)):
```

Comment:
extremely pedantic: this is the A variant so this is ASCII not utf-8

Reply:
im sorry for even saying anything

Reply:
Oh no man, you're good. Pedantic is fine. I'd rather it be brought up so we can talk about it and make sure all is good, rather than skipping past it.

And yea. This is the A variant, but if a pure ascii string doesn't decode to the same byte values in utf-8, I'm going to be sad for so many other reasons than this. So I'm no too worried about this particular one.